src/screens/Home/HomeV2.res

```diff
@@ -351,7 +343,7 @@ module Resources = {
     <>
       <div className="flex flex-col gap-4">
         <p className=headingStyle> {"Resources"->React.string} </p>
-        <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
+        <div className="flex flex-row gap-4">
```

Comment:
Ensure the layout is optimized for screens smaller than md and that the boxes are properly aligned.
<img width="597" alt="image" src="https://github.com/user-attachments/assets/a130932b-6568-4444-a4ed-9046fed7e5ec">